testsuite/tests/ui/mail_and_messages/test_account_messages.py

```diff
@@ -1,12 +1,33 @@
 """Test of automatic email associated to accounts in UI"""
 import re
 
+import pytest
+
+from testsuite import rawobj
 from testsuite.config import settings
 from testsuite.ui.views.admin.foundation import BaseAdminView
 from testsuite.ui.views.admin.login import RequestAdminPasswordView, LoginView, ResetPasswordView
 from testsuite.ui.views.devel import Navbar
 from testsuite.ui.views.devel.login import ForgotPasswordView, LoginView as DevelLoginView
-from testsuite.utils import randomize
+from testsuite.utils import randomize, blame
+
+
+@pytest.fixture(scope="module")
+def account(custom_account, request, account_password):
+    """
+    Account scoped for module (rather than session) because account password is updated in this module.
+    """
+    iname = blame(request, "id")
+    account = rawobj.Account(org_name=iname, monthly_billing_enabled=False, monthly_charging_enabled=False)
+    account.update(
```

Comment:
Maybe better to create an Account with params than update it.

Reply:
This is how we use it in main conftest